test/Microsoft.Azure.SignalR.Common.Tests/RestClients/HealthApiFacts.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.SignalR.Tests;
+using Microsoft.Rest;
+using Xunit;
+
+namespace Microsoft.Azure.SignalR.Common.Tests.RestClients
+{
+    public class HealthApiFacts
+    {
+        [Fact]
+        public async void IsServiceHealthyReturnTrue()
+        {
+            var signalRServiceRestClient = new MockRestClientHelper()
```

Comment:
try a `class TestRestClient:SignalRServiceRestClient` with `response`  or `stauscode` passed in to ctor, code would be similar to:
```cs
using (var client = new TestRestClient(HttpStatusCode.OK))
{
    var healthApi = new HealthApi(client );
    ...
}

```